.github/CONTRIBUTING.rst

```diff
@@ -34,17 +34,18 @@ our workflow that are not covered by a bot or status check are:
 - All discussions that are not directly related to the code in the pull request
   should happen on `GitHub Issues <https://github.com/python/cpython/issues>`_.
 - Upon your first non-trivial pull request (which includes documentation changes),
-  feel free to add yourself to ``Misc/ACKS``
+  feel free to add yourself to `Misc/ACKS <https://github.com/python/cpython/blob/main/Misc/ACKS>`_.
 
 
 Setting Expectations
 --------------------
-Due to the fact that this project is entirely volunteer-run (i.e. no one is paid
-to work on Python full-time), we unfortunately can make no guarantees as to if
+Due to the fact that this project is primarily volunteer-run,
+we unfortunately can make no guarantees as to if
```

Comment:
I understand where you're coming from but I believe even the developers in residence have non-CPython responsibilities.

```suggestion

Due to the fact that this project is volunteer-run,
we unfortunately can make no guarantees as to if
```